docs/config/global.md

```diff
@@ -43,7 +43,7 @@ Whether to enable package download, default is true. 【Not recommended especial
 
 Any content you want which are automatically placed at the beginning of the head of the web page
 
-- How to configure PWA（Desktop、Android、IOS）：**[alist/issues/6724](https://github.com/alist-org/alist/issues/6724#issuecomment-2220251541)**
+- How to configure PWA（Desktop、Android、IOS）：**[OpenList/issues/6724](https://github.com/OpenListTeam/OpenList/issues)**
```

Comment:
Cannot simply replace this, the original issues contains instructions and need more investigations.